### PR TITLE
Restful

### PR DESCRIPTION
**Pull request by nzomkxia**
reconstruction dubbo admin, based on Vuetify and spring boot

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ The following modules in [Apache Dubbo(incubating)](https://github.com/apache/in
 You can get a release of dubbo monitor in two steps:
 
 #### dubbo admin
-dubbo admin is a spring boot application, you can start it with fat jar or in IDE directly
+please refer to the REAME.md in dubbo admin module
```
Comment (htynkn) [inline README.md]: typo.  should be `README`

File: dubbo-admin/dubbo-ops-backend/src/main/java/org/apache/dubbo/admin/domain/ServiceDO.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
+package org.apache.dubbo.admin.domain;
+
+/**
+ * @author zmx ON 2018/7/20
```
Comment (htynkn) [inline dubbo-admin/dubbo-ops-backend/src/main/java/org/apache/dubbo/admin/domain/ServiceDO.java]: `@author` is not suggested to include in source code

File: dubbo-admin/dubbo-ops-backend/src/main/java/org/apache/dubbo/admin/controller/ServiceController.java [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * @author zmx ON 2018/7/31
```
Comment (lovepoem) [inline dubbo-admin/dubbo-ops-backend/src/main/java/org/apache/dubbo/admin/controller/ServiceController.java]: remove author

File: dubbo-admin/dubbo-ops-frontend/src/components/ServiceDetail.vue [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+
+            }
+          ]
+      },
+      consumerDetails: {
+        type: Array,
+        default: () =>
+          [
+            {
+              ip: '192.168.1.3',
```
Comment (htynkn) [inline dubbo-admin/dubbo-ops-frontend/src/components/ServiceDetail.vue]: I don't have experience in `vue.js`, so I may wrong here. but this block looks like some test data? is it a right place to include it ?

**Comment by htynkn** [comment]
I would like to double confirm about i18n. with this PR, do we still have i18n support?

File: dubbo-admin/dubbo-ops-backend/src/main/java/org/apache/dubbo/admin/domain/ServiceDO.java (hunk 2 above)
Comment (diecui1202) [reply dubbo-admin/dubbo-ops-backend/src/main/java/org/apache/dubbo/admin/domain/ServiceDO.java]: Also need LICENSE info

File: dubbo-admin/dubbo-ops-frontend/src/components/ServiceDetail.vue (hunk 4 above)
Comment (nzomkxia) [reply dubbo-admin/dubbo-ops-frontend/src/components/ServiceDetail.vue]: it's the default value for test purpose

**Comment by nzomkxia** [comment]
@htynkn there's no i18n support yet

File: dubbo-admin/dubbo-ops-backend/src/main/java/org/apache/dubbo/admin/domain/ServiceDO.java (hunk 2 above)
Comment (nzomkxia) [reply dubbo-admin/dubbo-ops-backend/src/main/java/org/apache/dubbo/admin/domain/ServiceDO.java]: thanks，I'll remove author info and add license

File: dubbo-admin/dubbo-ops-backend/src/main/java/org/apache/dubbo/admin/registry/common/route/RouteRule.java [hunk 5]
```diff
@@ -14,21 +14,17 @@
...
+
+import org.apache.dubbo.admin.registry.common.domain.Route;
+import com.alibaba.dubbo.common.utils.StringUtils;
 
 import java.text.ParseException;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Map;
+import java.util.*;
```
Comment (lovepoem) [inline dubbo-admin/dubbo-ops-backend/src/main/java/org/apache/dubbo/admin/registry/common/route/RouteRule.java]: .* is not a good import style

File: dubbo-admin/dubbo-ops-backend/src/main/java/org/apache/dubbo/admin/controller/ServiceController.java [hunk 6]
```diff
@@ -0,0 +1,102 @@
+package org.apache.dubbo.admin.controller;
```
Comment (ningyu1) [inline dubbo-admin/dubbo-ops-backend/src/main/java/org/apache/dubbo/admin/controller/ServiceController.java]: this file without ` ASF ` License

File: dubbo-admin/dubbo-ops-backend/src/main/java/org/apache/dubbo/admin/domain/ServiceDO.java [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.apache.dubbo.admin.domain;
+
+public class ServiceDO {
```
Comment (ningyu1) [inline dubbo-admin/dubbo-ops-backend/src/main/java/org/apache/dubbo/admin/domain/ServiceDO.java]: Implements Serializable， Increase the property field ` serialVersionUID `，Overriding hashcode, equals method

File: dubbo-admin/dubbo-ops-backend/src/main/java/org/apache/dubbo/admin/domain/ServiceDetailDO.java [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+ */
+
+package org.apache.dubbo.admin.domain;
+
+import org.apache.dubbo.admin.registry.common.domain.Consumer;
+import org.apache.dubbo.admin.registry.common.domain.Provider;
+
+import java.util.List;
+
+public class ServiceDetailDO {
```
Comment (ningyu1) [inline dubbo-admin/dubbo-ops-backend/src/main/java/org/apache/dubbo/admin/domain/ServiceDetailDO.java]: Implements Serializable， Increase the property field ` serialVersionUID `，Overriding hashcode, equals method

File: dubbo-admin/dubbo-ops-backend/pom.xml [hunk 9]
```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ningyu1) [inline dubbo-admin/dubbo-ops-backend/pom.xml]: this file without ASF License

File: dubbo-admin/dubbo-ops-backend/src/main/resources/application.properties [hunk 10]
```diff
@@ -0,0 +1,3 @@
+dubbo.registry.address=zookeeper://127.0.0.1:2181
```
Comment (ningyu1) [inline dubbo-admin/dubbo-ops-backend/src/main/resources/application.properties]: this file without ASF License

File: dubbo-admin/dubbo-ops-frontend/pom.xml [hunk 11]
```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ningyu1) [inline dubbo-admin/dubbo-ops-frontend/pom.xml]: this file without ASF License

File: dubbo-admin/pom.xml [hunk 12]
```diff
@@ -3,13 +3,17 @@
 	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (ningyu1) [inline dubbo-admin/pom.xml]: this file without ASF License

**Pull request merged**